DaoBuilder/dcntDAOConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { DecentDAOConfig } from "./types";
+
+export const decentDAOConfig: DecentDAOConfig = {
+  lockStart: Math.floor(Date.now() / 1000), // (Now) Start time of token lock
+  lockDuration: 60 * 60 * 24 * 365, // 1 year
+  snapshotURL: "https://snapshot.org/#/decentdao.eth",
+  initialSupply: "1000000",
```

Comment:
initial supply will need to account for the 18 decimal places, so I think we need to add 18 0s here

https://eth-converter.com/ (which only works because Eth also has 18 decimal places)

Reply:
yeah I left it as just primtive values here but `parseEther` is used when this is called

Reply:
oh, @Da-Colon we can use `ethers.utils.parseEther("XXX")` here

Reply:
Yeah and looks like this is missing some zeroes (even without the 18 decimals added). Should be 1 billion tokens right? Could use the javascript integer underscore syntax and make this 1_000_000_000 for readability.